github/misc.go

```diff
@@ -195,3 +197,34 @@ func (c *Client) ListServiceHooks() ([]*ServiceHook, *Response, error) {
 
 	return *hooks, resp, err
 }
+
+var (
+	githubRE = regexp.MustCompile(`https?://(www\.)?github\.com/`)
```

Comment:
Thank you so much! The new version that uses `url.Parse` instead of regexp is so so so much nicer IMO! It's easier to read, understand, and verify. And it's probably faster. (I am really not a fan of regexpes. Especially in places where more standard tools can be used to parse a well known spec.)